ARCHITECTURE.md

```diff
@@ -117,6 +117,7 @@ Update instructions:
 | rerun_c   | Rerun C SDK                          |
 | rerun_py  | Rerun Python SDK                     |
 | re_sdk    | Rerun logging SDK                    |
+<!-- | re_renderer_examples    | Examples for re_renderer | -->
```

Comment:
Hmm, what's this?

Reply:
our ci wants now all crates to be listed here, but I figured that's a weird one to list. So I pretend list it!